cfgov/cfgov/settings/base.py

```diff
@@ -501,3 +501,8 @@
 
 # Akamai
 ENABLE_AKAMAI_CACHE_PURGE = os.environ.get('ENABLE_AKAMAI_CACHE_PURGE', False)
+if ENABLE_AKAMAI_CACHE_PURGE:
+    AKAMAI_USER = os.environ['AKAMAI_USER']
```

Comment:
@higs4281 I tend to agree with you but @chosak brought the point up that if the config variables were not set, this would quietly fail (https://github.com/cfpb/cfgov-refresh/pull/2417#discussion_r80318403). This could be a reason to call `os.environ` directly from the function which I was doing earlier, but there also seems to be a preference for keeping these in `settings/base.py`.